src/js/jsx/sections/layers/LayersPanel.jsx

```diff
@@ -550,8 +551,7 @@ define(function (require, exports, module) {
                         }
                     }, this)
                     .map(function (layer, visibleIndex) {
-                        var isDragTarget = dragTargets && dragTargetSet.has(layer),
-                            isDropTarget = dropTarget && dropTarget.key === layer.key,
+                        var isDropTarget = !!dropTarget && dropTarget.key === layer.key,
```

Comment:
Adding the `!!` will make sure `isDropTarget` is always a boolean value, this is the cause of the performance issue when dragging over dragged items. So, before you drag over the dragged items, the `dropTarget` has value and `isDropTarget ` is false for all non-drop targets, but when you drag over the dragged items, `dropTarget` changed to null, so as the `isDropTarget`, and cause all non-drop targets to do a re-render because `this.props.isDropTarget !== nextProps.isDropTarget`.